modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/search_bar.js

```diff
@@ -16,6 +16,10 @@ AUI.add(
 	'liferay-search-bar',
 	A => {
 		var SearchBar = function(form) {
+			if (!form) {
+				return;
+			}
```

Comment:
Can we change it so we wrap the entire function in `if (form)` instead of returning early?

Reply:
Sure! There's a few places where this is written that way, I can update those too.

Reply:
@jonmak08 I actually like the early return in this case since it means one less level of nesting that seems unnecessary. There was a similar discussion about this too with allowing multiple returns and it seems other frontend devs like the early return pattern too. https://github.com/liferay/liferay-frontend-guidelines/issues/6

Since this is just a null check, I think wrapping a large block of code in an `if` statement makes it less readable so I'd rather keep this as is.

Reply:
Ok if thats the case, then we can side with having it `return` early.  I did not know that was a change we made for our formatter.  Good to know.  Thanks for pointing that out @thektan!